thoth/package_extract/core.py

```diff
@@ -28,8 +28,11 @@
 from .image import construct_rootfs
 from .image import download_image
 from .image import run_analyzers
+from prometheus_client import CollectorRegistry, pushadd_to_gateway, Gauge
 
 _LOGGER = logging.getLogger(__name__)
+prometheus_registry = CollectorRegistry()
+_METRIC_ANALYZER_JOB = Gauge('package_extract_time','Runtime of package extract job', registry=prometheus_registry)
```

Comment:
Let's reduce number of global variables - you can put ``prometheus_registry`` and ``_METRIC_ANALYZER_JOB`` (no need to have it uppercase in that case)  to the ``extract_image`` function where they are used.

Reply:
Alright, sure!